system/security/securesocket/securesocket.cpp

```diff
@@ -1284,47 +1393,30 @@ class CSecureSocketContext : implements ISecureSocketContext, public CInterface
             SSL_CTX_set_default_passwd_cb(m_ctx, pem_passwd_cb);
         }
 
-        const char* certfile = config->queryProp("certificate");
-        if(certfile && *certfile)
-        {
-            if (SSL_CTX_use_certificate_chain_file(m_ctx, certfile) <= 0)
-            {
-                char errbuf[512];
-                ERR_error_string_n(ERR_get_error(), errbuf, 512);
-                throw MakeStringException(-1, "error loading certificate chain file %s - %s", certfile, errbuf);
-            }
-        }
+        const char *certFileOrBuf = config->queryProp("certificatebuf");
```

Comment:
As mentioned above, I think we should rename these "buf" properties to "certificate_pem" or similiar so we know exactly what they are.  For any new formats in the future we can use other names.

Reply:
ok, renamed.